### PR TITLE
OrthographicFrustum deprecated properties are removed.

### LINKED ISSUE #5109: Remove deprecated OrthographicFrustum properties
Remove `'left`, 'right`, `bottom` and `top` properties on `OrthographicFrustum`.

### PR DESCRIPTION
**Pull request by ateshuseyin**
fixes #5109

### CODE REVIEW

**Comment by hpinkos** [comment]
Thanks @ateshuseyin!  We don't want to merge this until the 1.33 release, so we'll wait until after the 1.32 release goes out on Monday.

@bagnell does this look good to you?

**Comment by bagnell** [comment]
Can you also remove all references to the `_useDeprecated` property as well as the required `deprecationWarning`? Otherwise, this looks good.

File: CHANGES.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Change Log
 * Fixed issues with imagerySplitPosition and the international date line in 2D mode. [#5151](https://github.com/AnalyticalGraphicsInc/cesium/pull/5151)
 * Fixed an issue with `TileBoundingBox` that caused the terrain to disappear in certain places [4032](https://github.com/AnalyticalGraphicsInc/cesium/issues/4032)
 * `QuadtreePrimitive` now uses `frameState.afterRender` to fire `tileLoadProgressEvent` [#3450](https://github.com/AnalyticalGraphicsInc/cesium/issues/3450)
+* Removed left, right, bottom and top properties from `OrthographicFrustum`.
```
Comment (emackey) [inline CHANGES.md]: Move this up out of 1.32 changes.

**Comment by biomassives** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  acmeideal@gmail.com
    Domain biomassiv.es has exceeded the max emails per hour (106/100 (106%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext3.iad.github.net ([192.30.252.194]:43854 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by chi-server32.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.88)
	(envelope-from <noreply@github.com>)
	id 1cy1L2-000BId-D8
	for greg@biomassiv.es; Tue, 11 Apr 2017 14:20:28 -0500
Date: Tue, 11 Apr 2017 12:19:49 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1491938389;
	bh=CBxneXDKCbQet/dyGjXuqsHeq7Z3Gf0S/d+bHd+2G7E=;
	h=From:Reply-To:To:Cc:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=MhVOwJj4q32D4ZcFdvQKYlyI7hW36fBSM0dlenMICHAILhF/+ElWnmoHk/t8qooFq
	 sboNhshPaSIlxfgWFyqqnf73EUYrA45IlVMIm85u6rdzdwKJDaB27uqGy6y7urOetX
	 2pqRg6F9Fr+QARnRDxsuGDsG4/8SzmIByx6NVQCM=

**Comment by pjcozzi** [comment]
Thanks for updating CHANGES.md, @ateshuseyin.  Please comment on a PR after updating so the reviewers get notified.

@emackey please merge when ready.

**Comment by mramato** [comment]
@emackey is this good to go?  If so, I can just reserve the conflict in CHANGES and merge.  Thanks.

**Comment by emackey** [comment]
No, sorry, there are test failures here, on the removed properties.

File: Source/Scene/OrthographicFrustum.js [hunk 2]
```diff
@@ -86,15 +78,6 @@ define([
             frustum._near = frustum.near;
             frustum._far = frustum.far;
 
-            if (!frustum._useDeprecated) {
-                var ratio = 1.0 / frustum.aspectRatio;
```
Comment (bagnell) [inline Source/Scene/OrthographicFrustum.js]: You need to keep these statements because `frustum._useDeprecated` would always be `false`.

**Comment by ateshuseyin** [comment]
@emackey there are test failures and I cannot find the right solution. Could you help me?

**Comment by emackey** [comment]
@ateshuseyin Check out the [Testing Guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/TestingGuide/README.md) if you haven't run the tests yourself yet.

The source code for the tests is in the `Specs` folder, and its structure mostly matches the main `Source` folder.  Running the tests should indicate the filenames of the broken tests.  I think they're broken just because they're testing for the presence of deprecated values that you've removed from this branch.  So in that case, it's correct to remove the offending tests, as long as they're not testing anything else.

**Comment by emackey** [comment]
@ateshuseyin Oops, my previous comment was incorrect (so I deleted it).

Check out the code that @bagnell pointed out.  You removed a whole `if` block, one that should evaluate to `true` now.  If you restore the body of that block, the tests should pass.

**Comment by ateshuseyin** [comment]
@emackey and @bagnell thank you for your help. I reverted if block w/o `if(!this._useDeprecated)`.

**Comment by emackey** [comment]
Thanks @ateshuseyin.

@bagnell tests pass now.  Please merge when ready.

**Comment by pjcozzi** [comment]
Thanks again @ateshuseyin!

**Pull request merged**